### PR TITLE
ICP-3923 GoControl Integrations

### PR DESCRIPTION
**Pull request by greens**

### CODE REVIEW

File: devicetypes/smartthings/zwave-motion-sensor.src/zwave-motion-sensor.groovy [hunk 1]
```diff
@@ -30,6 +30,7 @@ metadata {
 		fingerprint mfr: "0060", prod: "0001", model: "0003", deviceJoinName: "Everspring Motion Sensor"  // Everspring HSP02
 		fingerprint mfr: "011A", prod: "0601", model: "0901", deviceJoinName: "Enerwave Motion Sensor"  // Enerwave ZWN-BPC
 		fingerprint mfr: "0063", prod: "4953", model: "3133", deviceJoinName: "GE Portable Smart Motion Sensor"
+		fingerprint mfr: "014F", prod: "2002", model: "0203", deviceJoinName: "Linear PIR Motion Sensor"
```
Comment (dkirker) [inline devicetypes/smartthings/zwave-motion-sensor.src/zwave-motion-sensor.groovy]: So... The device supports temperature. I noticed that it works fine with the Z-Wave Plus Motion/Temp Sensor, if state.sec gets set to 0 (it defaults to secure if state.sec is undefined -- I tested with an installed() that set state.sec-0). Maybe we could not use state.sec and just read the z-wave raw data for 's' and use this fingerprint with that DTH?

cc: @mckeed

Comment (greens) [reply devicetypes/smartthings/zwave-motion-sensor.src/zwave-motion-sensor.groovy]: Yeah that's the old method of checking security I think. It's what got us in trouble with the Fibaro Smoke Sensor.

**Comment by greens** [comment]
Moved the PIR sensor and updated the security handling.

**Comment by greens** [comment]
@dkirker ping

**Review by dkirker — APPROVED** [review]

**Comment by workingmonk** [comment]
the non-fingerprint change is in zwave-plus-motion-temp-sensor.groovy

**Pull request merged**